frontend/scss/pages/_p-issuearticle-show.scss

```diff
@@ -412,6 +451,10 @@
         background-image: unset !important;
       }
 
+      &.m-media--360-embed{
+        max-height: unset !important;
```

Comment:
SonarCloud is complaining about this duplicate selector:
https://sonarcloud.io/project/issues?sinceLeakPeriod=true&issueStatuses=OPEN%2CCONFIRMED&pullRequest=598&id=art-institute-of-chicago_artic.edu&open=AZMsBJVLjO3B_bnNS4EN

Could you move this line up to 433?